client/webserver/site/src/js/mmsettings.ts

```diff
@@ -44,116 +60,116 @@ const arbMMRowCacheKey = 'arbmm'
 const specLK = 'lastMMSpecs'
 const lastBotsLK = 'lastBots'
 const lastArbExchangeLK = 'lastArbExchange'
-const tokenFeeMultiplier = 50
-
-const qcDefaultProfitThreshold = 0.02
-const qcDefaultLevelSpacing = 0.005
-const qcDefaultMatchBuffer = 0
-const qcDefaultLotsPerLevelIncrement = 100 // USD
-const qcDefaultLevelsPerSide = 1
-const qcDefaultProfitMax = 0.1
-const qcMinimumProfit = 0.001
-
-const driftToleranceRange: XYRange = {
-  start: {
-    label: '0%',
-    x: 0,
-    y: 0
-  },
-  end: {
-    label: '1%',
-    x: 0.01,
-    y: 1
-  },
-  xUnit: '',
-  yUnit: '%'
-}
 
-const orderPersistenceRange: XYRange = {
-  start: {
-    label: '0',
-    x: 0,
-    y: 0
-  },
-  end: {
-    label: 'max',
-    x: 21,
-    y: 21
-  },
-  xUnit: '',
-  yUnit: 'epochs'
+const defaultSwapReserves = {
+  n: 50,
+  prec: 0,
+  inc: 10,
+  minR: 0,
+  maxR: 1000,
+  range: 1000
 }
-
-const oracleBiasRange: XYRange = {
-  start: {
-    label: '-1%',
-    x: -0.01,
-    y: -1
-  },
-  end: {
-    label: '1%',
-    x: 0.01,
-    y: 1
-  },
-  xUnit: '',
-  yUnit: '%'
+const defaultOrderReserves = {
+  factor: 1.0,
+  minR: 0,
+  maxR: 2,
```

Comment:
I would allow a little extra.. maybe 3x?

Reply:
No problem. While I'm here, I'm going to remove order reserves from basic-arb and add it to basic-mm. Basic-arb should never reserve anything if it can use it in a profitable arb.